### ISSUE TITLE
Integration of other languages like PHP

### ISSUE THREAD

**Issue by ichiriac** [fkling/astexplorer#205]
Hi,

I like a lot the result, I mean it's awesome to hover the node and view the code highlighted. I'm working on a [parser for php](https://github.com/glayzzle/php-parser) and  I would like to use it on my demo page.

Would be interested by a PR ?

**Comment by fkling** [comment]
Yes, the more parsers the better! 😃

**Comment by wmhilton** [comment]
Ah crap! This is why I need to read all the issues before starting a project. :) @ichiriac have you started on this? If so we should share code so our efforts aren't duplicated.

**Comment by wmhilton** [comment]
@ichiriac Got it working! https://github.com/wmhilton-contrib/astexplorer I made you a collaborator so you can push changes and we can make a PR.

p.s. I added a PHP svg icon to the folder, but didn't feel like going through the effort of installing this font-customizer thingy, so I don't actually know if the PHP logo looks right. I'm tempted to suggest abandoning the font thing and just inserting SVGs. Or using a class applied to an `<i>` that sets the background with an SVG. IDK. Anyway, point is there is a blank spot where the PHP icon should be, but I did add a PHP logo so if you run the fontcustom thing it might appear miraculously.

**Comment by ichiriac** [comment]
@wmhilton many thanks, did not have time yet to start something

**Comment by ichiriac** [comment]
closing this issue, as it was already done here : https://github.com/fkling/astexplorer/pull/253

**Comment by wmhilton** [comment]
Awesome!!!
